gradle.properties

```diff
@@ -1,4 +1,4 @@
 cdvCompileSdkVersion=30
 cdvMinSdkVersion=23
 android.useAndroidX=true
-android.enableJetifier=true
+android.enableJetifier=false
```

Comment:
Can we remove this line? I assume the default is `false`.

Reply:
Also, does this work for a Cordova-generated project?

Reply:
I wanted to be really explicit, so I was thinking of just setting it to `false`. But I can certainly remove it.

I don't know about cordova - I can say that all automated tests are passing.

Reply:
Let's just remove it :-)